<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>661</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>661</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Segoe UI&quot;;
background-color: rgb(217, 233, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>425</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#saveButton {
    background-color: rgb(24, 11, 170);
    font: 700 8pt &quot;Segoe UI&quot;;
    color: rgb(255, 255, 255);
    border-radius: 6px;
    border: 1px solid rgb(24, 11, 170); 
}

QPushButton#saveButton:pressed {
    background-color: #3498db; 
}
</string>
   </property>
   <property name="text">
    <string>Save changes</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>425</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#addButton {
    background-color: rgb(24, 11, 170);
    font: 700 8pt &quot;Segoe UI&quot;;
    color: rgb(255, 255, 255);
    border-radius: 6px;
    border: 1px solid rgb(24, 11, 170); 
}

QPushButton#addButton:pressed {
    background-color: #3498db; 
}
</string>
   </property>
   <property name="text">
    <string>Add Task</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>661</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(69, 2, 127);

color: rgb(255, 255, 255);
font: 24pt &quot;Segoe UI&quot;;
border-radius:8px;</string>
   </property>
   <property name="text">
    <string>Task Manager</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="taskLineEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>420</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-top-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeButton">
   <property name="geometry">
    <rect>
     <x>448</x>
     <y>425</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#removeButton {
    background-color: rgb(24, 11, 170);
    font: 700 8pt &quot;Segoe UI&quot;;
    color: rgb(255, 255, 255);
    border-radius: 6px;
    border: 1px solid rgb(24, 11, 170); 
}

QPushButton#removeButton:pressed {
    background-color: #3498db; 
}
</string>
   </property>
   <property name="text">
    <string>Remove Selected Task</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>460</y>
     <width>661</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../Downloads/bbbb.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
